php/classes/integrations/elementor/widgets/elementor-episode-list-widget.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace SeriouslySimplePodcasting\Integrations\Elementor\Widgets;
+
+use SeriouslySimplePodcasting\Controllers\Episode_Controller;
+use WP_Query;
+
+class Elementor_Episode_List_Widget extends \Elementor\Widget_Base {
+	public function get_name() {
+		return 'Episode List';
+	}
+
+	public function get_title() {
+		return __( 'Episode List', 'seriously-simple-podcasting' );
+	}
+
+	public function get_icon() {
+		return 'fa fa-list';
+	}
+
+	public function get_categories() {
+		return [ 'podcasting' ];
+	}
+
+	public function get_episodes() {
+		$args = array(
+			'fields'         => array( 'post_title, id' ),
+			'posts_per_page' => - 1,
```

Comment:
This is an interesting one. Ideally, we should never use -1 as the posts_per_page value, if the user has thousands of episodes, this query could crash WordPress on shared web hosting. 

WordPress has a built in option in the options table called posts_per_page, which the user can set from in the Settings -> Reading Settings area, so it would probably be a good idea to use this value, or default to 10 if this option doesn't exist for any reason.

`'posts_per_page' => get_option('posts_per_page', 10),`